ynab.gemspec

```diff
@@ -5,10 +5,10 @@
 
 #Our API uses a REST based design, leverages the JSON data format, and relies upon HTTPS for transport. We respond with meaningful HTTP response codes and if an error occurs, we include error details in the response body.  API Documentation is at https://api.ynab.com
 
-The version of the OpenAPI document: 1.0.0
+The version of the OpenAPI document: 1.69.0
 
 Generated by: https://openapi-generator.tech
-OpenAPI Generator version: 6.6.0
+Generator version: 7.6.0
```

Comment:
Did you install `openapi-generator` with brew? I think I overlooked documenting this needed step in https://github.com/ynab/ynab-sdk-ruby/blob/main/DEVELOPMENT.md.  Can we add a mention for this there, to install and/or update `openapi-generator`?

Reply:
Sure thing, done with 2fcf5135f13720c37127194fb7a7f3d33550f5b8